PhysicsTools/PatAlgos/runMiniAOD_90X.py

```diff
@@ -0,0 +1,134 @@
+# Auto generated configuration file
```

Comment:
please remove this file:
it should be either in /test directory or not present at all
miniAOD processing is exercised already in many matrix workflows